js/notify/electron-notify-preload.js

```diff
@@ -134,6 +136,37 @@ function setContents(event, notificationObj) {
 
     const winId = notificationObj.windowId;
 
+    let displayTime = notificationObj.displayTime || 5000;
```

Comment:
why move this here?  would rather not - we are separating logic from electron-notify.

Reply:
@lneir In order not close the notification when the mouse is over it we need access to `mouseleave` `mouseover` events so I had to move this logic to preload

Reply:
Done 😃 I've updated the logic. Please review. Thanks 😃